src/MongoDB/Manager.c

```diff
@@ -916,6 +924,41 @@ static HashTable* php_phongo_manager_get_debug_info(zval* object, int* is_temp T
 	mongoc_server_descriptions_destroy_all(sds, n);
 
 	return Z_ARRVAL(retval);
+
+failure:
```

Comment:
I think we still want to do `mongoc_server_descriptions_destroy_all(sds, n)` here, lest those leak.